code/controllers/verbs.dm

```diff
@@ -76,6 +75,30 @@
 		if("Vote")
 			debug_variables(vote)
 			feedback_add_details("admin_verb","DVote")
+		if("Diseases")

```

Comment:
Something can be said about polluting the choices list, but I suppose it's fair enough.